### ISSUE TITLE
[Extension] Extension will not run if WALA use customerized configuration file path

### ISSUE THREAD

**Issue by yuhuijiang** [Azure/azure-linux-extensions#487]
If we use a customerized configuration file path to WALA, extension VMAccessForLinux will not successfully install.Because inside VMAccessForLinux extension,it will only check the default configuration file path(/etc/waagent.conf).If /etc/waagent.conf not exist,it will unistall.

**Comment by boumenot** [comment]
Correct, VMAccessForLinux does not support containerization at this time.

**Comment by otubo** [comment]
@jasonzio @boumenot you guys are planning to release this feature soon? If there's no development on going I can step up and work on it.

**Comment by otubo** [comment]
Looks like the pull request #370 fixes the issue, maintainers could merge that so we could give it a try?
CC @yuxisun1217

**Comment by boumenot** [comment]
That is the correct PR to address the issue, but there was an internal discussion that caused us to not commit.  I am following up to see what the current opinion is.

**Comment by boumenot** [comment]
That's a great point!  Let me bring that up.

**Comment by otubo** [comment]
Actually other members of the project doesn't seem to agree on this matter, take a look at: https://github.com/Azure/WALinuxAgent/issues/1129

**Comment by otubo** [comment]
Actually other members of the project doesn't seem to agree on this matter, take a look at: https://github.com/Azure/WALinuxAgent/issues/1129

**Comment by boumenot** [comment]
I know.  You see only the external conversations, and that was before we had a change to debate the various ideas.  We did, and we are trying to come to a consensus on this.
